### PR TITLE
check for faucet-tags-in-forch early-on in kokoro/check.sh. add a scr…

### PR DESCRIPTION
**Pull request by pbatta**
…ipt to clean-up tags

### CODE REVIEW

File: kokoro/check.sh [hunk 1]
```diff
@@ -105,5 +105,17 @@ else
     echo Merge-base of $gtag and perry/master is $mtag
 fi
 
+#Check if faucet tags were added into forch
```
Comment (grafnu) [inline kokoro/check.sh]: Space after #

File: kokoro/check.sh [hunk 2]
```diff
@@ -105,5 +105,17 @@ else
...
+    faucet_tag=`git ls-remote $remote v1_0`
+    if [ -n "$faucet_tag" ]; then
+        echo
+        echo Error:
+        echo Faucet tag found in Forch repo: `git remote get-url $remote`
+        echo
+        false
+    fi
+done
+
```
Comment (grafnu) [inline kokoro/check.sh]: should also check local repo for tags

File: kokoro/check.sh [hunk 3]
```diff
@@ -105,5 +105,17 @@ else
     echo Merge-base of $gtag and perry/master is $mtag
 fi
 
+#Check if faucet tags were added into forch
+for remote in faucet origin perry; do
+    faucet_tag=`git ls-remote $remote v1_0`
+    if [ -n "$faucet_tag" ]; then
+        echo
+        echo Error:
+        echo Faucet tag found in Forch repo: `git remote get-url $remote`
```
Comment (grafnu) [inline kokoro/check.sh]: this is redundant with the output at the top of the file for the remote URL... I'd keep it as the semantic name and not the url

File: kokoro/clean_tags.sh [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        git push --delete $remote $tag
+    done
+done
+
+# clean up in local repo. delete all tags and fetch the right set
+git tag -d $(git tag)
+git fetch --tags
+
+# check if everything is clean
+faucet_tag=`git tag --list | fgrep 1.9.35`
```
Comment (grafnu) [inline kokoro/clean_tags.sh]: Maybe check the same tag as in the other script, just for consistency

File: kokoro/clean_tags.sh [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+if [ -n $faucet_tag ]; then
```
Comment (grafnu) [inline kokoro/clean_tags.sh]: Should have "$faucet_tag" for consistency when the variable is empty

File: kokoro/check.sh (hunk 1 above)
Comment (pbatta) [reply kokoro/check.sh]: fixed.

File: kokoro/check.sh (hunk 2 above)
Comment (pbatta) [reply kokoro/check.sh]: added, thanks.

File: kokoro/check.sh (hunk 3 above)
Comment (pbatta) [reply kokoro/check.sh]: done, changed to semantic name

File: kokoro/clean_tags.sh (hunk 4 above)
Comment (pbatta) [reply kokoro/clean_tags.sh]: done, v1_0 in both scripts now.

File: kokoro/clean_tags.sh (hunk 5 above)
Comment (pbatta) [reply kokoro/clean_tags.sh]: fixed, thanks.

**Review by grafnu — APPROVED** [review]

**Pull request merged**